src/main/steps/application-within-proceedings/help-with-fees/apply-for-hwf/content.ts

```diff
@@ -0,0 +1,77 @@
+import { TranslationFn } from '../../../../app/controller/GetController';
+import { FormContent } from '../../../../app/form/Form';
+import { getApplicationDetails } from '../../../../steps/application-within-proceedings/utils';
+import { getCasePartyType } from '../../../../steps/prl-cases/dashboard/utils';
+
+export const en = {
+  title: 'Apply for help with fees',
+  applyBefore: 'You must apply for help with fees before submitting your application.',
+  nextSteps: 'Next steps',
+  line1:
+    'Go to <a href="https://www.gov.uk/get-help-with-court-fees" class="govuk-link" rel="external" target="_blank">apply for help with fees (opens in a new tab)</a>',
+  line2: 'Enter',
+  enterCourtNumber: 'when you are asked to enter a court or tribunal number',
+  line3: 'Complete the help with fees application',
+  line4: 'Return to complete your',
+  applicationTo: 'application to',
+  line5: 'Enter your help with fees reference number',
+  onlyContinue: 'Continue',
+  cancel: 'Cancel',
+};
+
+export const cy: typeof en = {
+  title: 'Apply for help with fees (welsh)',
+  applyBefore: 'You must apply for help with fees before submitting your application. (welsh)',
+  nextSteps: 'Next steps (welsh)',
+  line1:
```

Comment:
Please use arrays for showing the steps. do not split lines.

Reply:
Changed to use arrays and whole lines